cmd/createtree/main.go

```diff
@@ -139,20 +139,12 @@ func newRequest(opts *createOpts) (*trillian.CreateTreeRequest, error) {
 func newPK(opts *createOpts) (*any.Any, error) {
 	switch opts.privateKeyType {
 	case "PEMKeyFile":
-		path := opts.pemKeyPath
-		if path == "" {
+		if opts.pemKeyPath == "" {
 			return nil, errors.New("empty PEM path")
 		}
-		if _, err := os.Stat(path); err != nil {
-			return nil, fmt.Errorf("error reading PEM key file at %v: %v", path, err)
-		}
-		pass := opts.pemKeyPass
```

Comment:
nit: Since we've kept the `path != ""` validation, would you mind keeping the `pass != ""` check too, for consistency?

Both checks seem reasonable to keep, as they inform users of missing flags.